### DISCUSSION TITLE
Create a Object Browser Filter for a List of source files

### DISCUSSION THREAD

**Discussion by thomprl** [codefori/vscode-ibmi#1713] [Ideas]
Would it be possible to create a Filter with a list of Objects instead of just one or a generic?  Something like below comma or semi colon separated so that I could pick just the source members that I want?  The way our change management system works it creates a large set of SRC-PF for different areas of our application when really I only need to look at a few. 

![image](https://github.com/codefori/vscode-ibmi/assets/17936568/26164e89-0386-4a66-8bc0-35d263e22c01)

**Comment by eraguive** [comment]
I was looking for the same thing.

We work with a set of source files that are into a "project", so I was trying to do this with a filter that accepts many criteria, but this is not currently possible.

**Comment by sebjulliand** [comment]
@thomprl , @aloaguilarv if you guys are interested, I implemented your suggestion. You can find a test build here in the comments if you want to give it a try: https://github.com/codefori/vscode-ibmi/pull/1780

**Reply by thomprl** [reply]
Thank you for implementing this.  I'm not sure how to give this a try. We have many source files created and this will really help.  Aldon our CMS creates every different source file known to man and then some in our development libraries.  But normally we only use just a few.  Just using Q* showed things like QMENUSRC and many more that aren't used that often.  Looking forward to this being implemented.

**Comment by thomprl** [comment]
I noticed this got released recently.  One thing that might need to be spelled out is that the comma-delimited list can not have spaces in it.  At first I entered QRPGLESRC, QDDSSRC, QCLSRC and it only pulled up the QRPGLESRC.  I don't know if it needs to trim the string or maybe the example needs to be added back?  ie Q* or * or QRPGLESRC,QDDSSRC,QCLSRC with no spaces.  Anyway thank you guys work working on this really makes life easier.

**Reply by sebjulliand** [reply]
Ugh, good catch! A PR is out to fix this, as whitespaces should not break the filtering process. See https://github.com/codefori/vscode-ibmi/pull/1848
